api/reporting/schemas/input/mysql_describe_csv/dataset.csv

```diff
@@ -8,4 +8,3 @@ invalid,bit(1),YES,"",,""
 includes_all_participants,bit(1),YES,"",,""
 last_modified_time,datetime,YES,"",,""
 version,smallint(6),NO,"",1,""
-pre_packaged_concept_set,tinyint(4),YES,"",,""
```

Comment:
The contract for how this all works will soon be simplified to a single yaml input file for each table. I'm moving away from storing this `EXPLAIN table` output and separate file for skipped rows, etc.